front/helpdesk.faq.php

```diff
@@ -57,7 +57,7 @@
 if (isset($_GET["id"])) {
     $kb = new KnowbaseItem();
     if ($kb->getFromDB($_GET["id"])) {
-        $kb->showFull();
+        $kb->showFull($_GET);
```

Comment:
It might be too permissive to put everything from `_GET` into the existing `$options` parameter.
Maybe only set the token ?

```suggestion
        $kb->showFull(['token' => $_GET['token'] ?? ""]);
```